common.tf

```diff
@@ -43,7 +43,8 @@ locals {
     local.rotate_preservation_system_password_name,
     local.ingest_start_workflow_lambda_name,
     local.ingest_upsert_archive_folders_lambda_name,
-    local.ingest_workflow_monitor_lambda_name
+    local.ingest_workflow_monitor_lambda_name,
+    local.ingest_queue_creator_name
```

Comment:
Should this have "lambda" in its name for consistency?

Reply:
It's also being used for the queue name which is why I didn't put lambda in there. If you look at the other lambdas that have queues like `local.copy_files_from_tdr_name` then they're the same.